README.md

```diff
@@ -3,6 +3,55 @@
 This project contains a Java-Builder for Exasol error messages.
 The invocations of the Builder can be parsed by the [error-code-crawler-maven-plugin](https://github.com/exasol/error-code-crawler-maven-plugin).
 
+## Usage
+
+### Simple Messages
+```java
+ExaError.messageBuilder("E-TEST-1").message("Something went wrong.").toString();
+```
+result: `E-TEST-1: Something went wrong.`
+
+### Parameters
+
```

Comment:
Maybe we can add a small one-liner explanation here about what are the parameters for.

Something like:
`You can specify placeholders in the messages and fill them up with parameters, as follows:`